lib/utils.js

```diff
@@ -30,3 +30,7 @@ module.exports.getRandomNumber = (max) => {
   let randomNumber = Math.floor(Math.random() * max) + 1;
   return randomNumber;
 };
+
+module.exports.stripFltSpecialChars = (string) => {
```

Comment:
"Flt" is a bit tough to follow here. Fluent uses the `.ftl` extension for it's files, so if you are going to abbreviate I'd use that instead. However, "fluent" is a short word. You can just include the whole thing here.

(Bonus trick, if you do this in VSCode, you can put your cursor on the function name, press F2, and type in a new name. It should rename it everywhere that references this export.)

Reply:
Hah so I was intending to put ftl... but I flipped the letters, I'm just switching it to the  full word so it's not an issue.